en/patterns/tree-grid.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Tree Grid - Design System Pattern
+_description: The Tree Grid Pattern Symbol represents a Tree Grid Component with sub-levels nested and features applied in a ready-to-use element.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Tree Grid
+
+Use the Tree Grid Pattern symbol as an alternative starting point when designing a tabular data layout with child and parent records within a Grid. It provides a full-blown contentwise and stylewise Tree Grid Component, on which can be applied all the Grid functionality, provided in the Components Library such as:
+
+- Sorting
+- Filtering
+- Summaries
+- Paging
+- Editing
+- Column Moving
+- Column Resizing
+- Column Hiding
+- Display Density
+- Excel Style Filtering
+- GroupBy
+- Pinning
+- Row Selection
+- Row Editing
+- Active Cell
+- Export
+- Toolbar
+
+All Grid features are applicable to the Tree Grid Pattern.
+
+<img class="responsive-img" src="../images/tree_grid.png" srcset="../images/tree_grid@2x.png 2x" />
```

Comment:
Image is pushed down a lot by the content ideally the image should be above the fold when the page loads. Move it up.